test/examples/ucp_hello_world.c

```diff
@@ -0,0 +1,16 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
+*
+* $COPYRIGHT$
+* $HEADER$
+*/
+
+#define HAVE_CONFIG_H
```

Comment:
The purpose of the test is to fail if config.h is used.
If HAVE_CONFIG_H is not defined, UCX's bad headers will still silently "skip" including config.h because usually it's wrapped in this `#ifdef`.